php-pre-challenge1/index.php

```diff
@@ -0,0 +1,12 @@
+<?php
+for ($i =1; $i <= 100; $i++){
+        if ($i % 15 === 0){
+            echo "3の倍数であり、5の倍数<br>";
+        }   elseif  ($i % 3 === 0){
+            echo "3の倍数<br>";
+        }   elseif  ($i % 5 === 0){
+            echo "5の倍数<br>";
+        }   else    {
```

Comment:
```suggestion
        } else {
```
トークンの間のスペースは他の行と揃えなくてよいです。

Reply:
了解です